models/Mall.js

```diff
@@ -3,7 +3,12 @@ const mongoose= require('mongoose')
 const mallSchema = mongoose.Schema({
     nameOfMall:String,
     aboutOfMall:String,
-    rating: Number,
+    rating: {
+        type: Number,
+        required: true,
```

Comment:
Will this affect the creation of new malls? on creating new mall there is no rating to be given.